src/DoctrineAuditBundle/Command/CleanAuditLogsCommand.php

```diff
@@ -45,7 +45,7 @@ protected function configure(): void
             ->setDescription('Cleans audit tables')
             ->setName(self::$defaultName)
             ->addOption('no-confirm', null, InputOption::VALUE_NONE, 'No interaction mode')
-            ->addArgument('keep', InputArgument::OPTIONAL, 'Keep last N months of audit.', '12')
+            ->addArgument('keep', InputArgument::OPTIONAL, 'Keep audits up to a date subtracted by a ISO 8601 date interval (e.g. P12M for 12 months or P7D for 7 days).', 'P12M')
```

Comment:
This sentence is not very clear to me, what about: "Audits retention period (must be expressed as an ISO 8601 date interval, e.g. P12M to keep the last 12 months or P7D to keep the last 7 days)." ?

Reply:
Sure. I like it.